### DISCUSSION TITLE
The puppeteer error while building

### DISCUSSION THREAD

**Discussion by atapas** [reactplay/react-play#112] [Q&A] [answered]
@erayalkis 

I am seeing this intermittent error in the log and it fails the build. Do you think, we have a fix for it? Starting this discussion, if you want to get to a root-cause, welcome. When you have a fix, let's convert this one to an issue.

![image](https://user-images.githubusercontent.com/3633137/165795580-af759258-2e9f-4473-a72c-ff71cfb2c475.png)

**Comment by erayalkis** [comment, marked as answer]
It looks like puppeteer is having trouble navigating to the Movies route. Since this component makes an API call, maybe it leaves puppeteer hanging when an error occurs?

**Comment by erayalkis** [comment]
A hacky workaround for this would be to disable the timeout feature on puppeteer. Perhaps we can start by testing this out .

**Reply by atapas** [reply]
Is there a way we can configure it anywhere? Like with a higher timeout value?

**Reply by erayalkis** [reply]
Since we can't access the pupeteer functions directly there must be some other way to change the timeout value, I'll look into it ASAP!

**Reply by erayalkis** [reply]
I've looked at every single source I could find and they either recommend downgrading to react-snap v 1.10 or upgrading puppeteer to the latest version. Maybe we could start by upgrading puppeteer?

**Reply by atapas** [reply]
Sounds like a plan.

**Reply by erayalkis** [reply]
Alrighty! I can create a PR updating the necessary packages if that sounds alright!

**Reply by atapas** [reply]
Sorry I forgot to respond. It is worth exploring. converting this as an issue
